tfe/testing.go

```diff
@@ -140,6 +140,22 @@ func createOrganization(t *testing.T, client *tfe.Client, options tfe.Organizati
 	}
 }
 
+func createOrganizationMembership(t *testing.T, client *tfe.Client, orgName string, options tfe.OrganizationMembershipCreateOptions) (*tfe.OrganizationMembership, func()) {
+	ctx := context.Background()
+	orgMembership, err := client.OrganizationMemberships.Create(ctx, orgName, options)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return orgMembership, func() {
+		if err := client.OrganizationMemberships.Delete(ctx, orgMembership.ID); err != nil {
+			t.Errorf("Error destroying organization membership! WARNING: Dangling resources\n"+
+				"may exist! The full error is show below:\n\n"+
+				"Organization memberships:%s\nError: %s", orgMembership.ID, err)
+		}
+	}
```

Comment:
I'm curious if we can get away with registering the cleanup function inside the helper rather than returning it:
```go
t.Cleanup(func() {
    // delete orgMembership
})
return orgMembership
```

Reply:
yes, this should work as well